rewrite-protobuf/src/test/kotlin/org/openrewrite/protobuf/JsonParserTest.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.protobuf
+
+import org.assertj.core.api.Assertions.assertThat
+import org.intellij.lang.annotations.Language
+import org.openrewrite.InMemoryExecutionContext
+import org.openrewrite.internal.StringUtils
+
+open class ProtoParserTest {
```

Comment:
Filename is `JsonParserTest`. Not a fatal error to kotlin, but possibly confusing to readers